aiida_common_workflows/workflows/relax/gpaw/generator.py

```diff
@@ -0,0 +1,151 @@
+# -*- coding: utf-8 -*-
+"""Implementation of `aiida_common_workflows.common.relax.generator.CommonRelaxInputGenerator` for GPAW."""
+from typing import Any, Dict, List, Tuple, Union
+import pathlib
+import yaml
+from aiida import engine
+from aiida import orm
+from aiida import plugins
+
+from aiida_common_workflows.common import ElectronicType, RelaxType, SpinType
+from ..generator import CommonRelaxInputGenerator
+
+__all__ = ('GPAWCommonRelaxInputGenerator',)
```

Comment:
We maintain standard Python naming convention of pascal case, so please use `Gpaw` instead of `GPAW` everywhere.
```suggestion
__all__ = ('GpawCommonRelaxInputGenerator',)
```